testcafe/tests/drive/folder_sharing_scenario.js

```diff
@@ -0,0 +1,130 @@
+import { Selector, Role } from 'testcafe'
+import { driveUser } from '../helpers/roles'
+import {
+  TESTCAFE_DRIVE_URL,
+  isExistingAndVisibile,
+  getCurrentDateTime,
+  FOLDER_DATE_TIME
+} from '../helpers/utils'
+import Data from '../helpers/data'
+const data = new Data()
+
+import DrivePage from '../pages/drive-model'
+const drivePage = new DrivePage()
+
+import PublicDrivePage from '../pages/drive-model-public'
+const publicDrivePage = new PublicDrivePage()
+
+const file = '[cozy]QA_table_ APPS.pdf'
```

Comment:
We should have a constant file with this kind of variables.

Reply:
Added in data.js
(and I removed the constructor)